ccloud-monitoring/docs/producer-scenarios/authorization-problem.rst

```diff
@@ -0,0 +1,59 @@
+.. _ccloud-monitoring-producer-authorization-problem:
+
+Authorization Revoked
+*********************
+
+Using the |ccloud| CLI, revoke the producer's authorization to write to the topic.
+
+This scenario will look at |ccloud| metrics from the Metrics API and client metrics from the client application's MBean object ``kafka.producer:type=producer-metrics,client-id=producer-1``.
+
+Introduce failure scenario
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. Delete the ACL that allowed the service account to write to the topic, inserting your service account ID instead of ``184498``:
+
+   .. code-block:: bash
+
+      ccloud kafka acl delete --service-account 184498 --operation write  --topic '*' --allow
+
+Diagnose the problem
+^^^^^^^^^^^^^^^^^^^^
+
+.. include:: ../includes/produce-failures.rst
+
+#. Check the producer logs for more information about what is going wrong. Use the following docker command to get the producer logs:
```

Comment:
I don't agree with adding all of the observations from produce-failures.rst here. When running this scenario locally, I don't see a change in performance metrics, retry rate, or buffer space. Below is a screenshot of the dashboard, at the time of the screenshot the producer lost access to the topic for ~30 minutes. I see the error you describe in the producer client logs. 
<img width="1727" alt="Screen Shot 2021-02-23 at 8 49 24 AM" src="https://user-images.githubusercontent.com/12937975/108869067-17df3800-75b4-11eb-9d4a-271005d30da0.png">

Reply:
Did you see something similar to this? I am surprised to not see an increase in failed authentication, nor a decrease in outgoing bytes. I would guess org.apache.kafka.common.errors.TopicAuthorizationException isn't a retriable exception which would explain why we don't see anything with the retry rate. That might be something to mention in this section?

Reply:
@awalther28 We discussed this over Zoom and I made modifications to the PR to reflect our understanding.  Please let me know what you think of the latest commit